src/Sender.cc

```diff
@@ -976,21 +983,49 @@ Sender::checkPingTimeouts()
 }
 
 /**
- * Send out packets for any messages with unscheduled/granted bytes.
+ * Attempt to wake up the pacer thread that is responsible for calling trySend()
+ * repeatedly, if it's currently blocked waiting for packets to become ready to
+ * be sent.
+ *
+ * This method is called when new GRANTs arrive, when new outgoing messages
+ * appear, and when retransmission is requested.
+ *
+ * @param lockHeld
+ *      Reminder to hold the Sender::queueMutex during this call.
  */
 void
-Sender::trySend()
+Sender::signalPacerThread(const SpinLock::Lock& lockHeld)
```

Comment:
Can we find a more general name for this method? It refers to the "pacer thread" concept which isn't documented, don't exists in this code base, and isn't necessarily used in all implementations.

Reply:
=> `signalSendReady`